users/models.py

```diff
@@ -65,9 +65,60 @@ def has_module_perms(self, app_label):
         return True
 
 
-class Wishlist(models.Model):
+class SavedPosts(models.Model):
     author = models.OneToOneField(MyUser, on_delete=models.CASCADE)
     post = models.ManyToManyField(Post)
-    
+
     def __str__(self):
         return f"Wishlist for {self.author.username}"
+
+
+class Profile(models.Model):
+    user = models.OneToOneField(MyUser, on_delete=models.CASCADE)
+    image = models.ImageField(upload_to='profile_pics', blank=True, null=True)
+    bio = models.TextField()
+    YEAR_CHOICES = (
+        ('1', 'First'),
+        ('2', 'Second'),
+        ('3', 'Third'),
+        ('4', 'Fourth'),
+    )
+    BRANCH_CHOICES = (
+        ('COE', 'Computer Engineering'),
+        ('CSE', 'Computer Science Engineering'),
+        ('ECE', 'Electroincs and Communications Engineering'),
+        ('ENC', 'Electroincs and Computer Engineering'),
+        ('COBS', 'Computer Science and Business Studies'),
+        ('CE', 'Chemical Engineering'),
+        ('CiE', 'Civil Engineering'),
+        ('BiE', 'Biotechnology'),
+        ('BME', 'Biomedical Engineering'),
+        ('SE', 'Structural Engineering'),
+        ('IE', 'Infrastructure Engineering'),
+        ('ME', 'Mechanical Engineering'),
+        ('MEC', 'Mechatronics Engineering'),
+        ('EE', 'Electrical Engineering'),
+        ('ECE', 'Electroinc (Instrumentation and Control)'),
+        ('ME(P)', 'Mechanical (Production) Engineering'),
+        ('BE-MBA(ME)', 'Mechanical MBA Dual Degree'),
+        ('BE-MBA(ECE)', 'Electroincs MBA Dual Degree'),
+        ('Others', 'Other'),
+    )
+    year = models.CharField(max_length=1, choices=YEAR_CHOICES, default='1')
```

Comment:
remove `default = '1'`
This input would be taken from user and not decided by default